lib/Models/Terria.ts

```diff
@@ -290,6 +290,10 @@ export interface ConfigParameters {
    * True to display in-app guides.
    */
   showInAppGuides?: boolean;
+  /**
+   * Show the "Search in Catalog" link in Location Search results
+   */
+  showSearchInCatalogLink?: boolean;
```

Comment:
Can this option be added to [SearchBarTraits](https://github.com/TerriaJS/terriajs/blob/main/lib/Traits/SearchProviders/SearchBarTraits.ts) instead? That' where other search bar config exists.

Then it can set in config.json as:
```json
{
  "searchBarConfig": {
     "showSearchInCatalog": false
  }
}
```